scripts/makepeds_psana

```diff
@@ -632,44 +632,62 @@ if [[ ( $HAVE_JUNGFRAU -ge 1 ) && ( $WANT_ZYLA -eq 0 ) && ( $WANT_OPAL -eq 0 ) &
         NJOBS=0
     fi
  
-    for JUNGFRAU in $DETNAMES; do
-        echo "jungfrau loop $JUNGFRAU  -- $RUNLOCAL"
+    for i in "${!DETNAMES[@]}"; do
+        JUNGFRAU="${DETNAMES[i]// /}"
+        ALIAS="${ALIASES[i]// /}"
+        echo "jungfrau loop $JUNGFRAU with alias $ALIAS -- $RUNLOCAL"
+        NSEG=99
+        if [[ $ALIAS =~ "5" ]]; then
+	    NSEG=1; 
+        elif [[ $ALIAS =~ "1M" ]]; then
+	    NSEG=2; 
+	elif [[ $ALIAS =~ "4M" ]]; then
+	    NSEG=8;
+	fi
         CMD="jungfrau_dark_proc $JFARG -s $JUNGFRAU"
         #now finally call the command
         for calibcycle in {0..2}; do
             if [ -v OLD_JUNGFRAU ]; then
                 CMDC=$CMD
-            elif  [ "$CALIBCODE" -eq 0 ]; then
+            elif [ "$CALIBCODE" -eq 0 ]; then
                 CMDC=$CMD' --stepnum '$calibcycle
             else
                 CMDC=$CMD
             fi
             if [[ $RUNLOCAL != 1 ]]; then
-                tmpScript=$(mktemp -p "$WORKDIR" jungfrau_multi_tmpXXXXX.sh)
-                #trap "rm -f $tmpScript" EXIT
-                chmod u+x "$tmpScript"
-                printf '#!/bin/bash\n' > "$tmpScript"
-                printf 'source %s/manage/bin/psconda.sh\n' "${SIT_ENV_DIR}" >> "$tmpScript"
-                printf 'echo $HOSTNAME\n' >> "$tmpScript"
-                printf '%s\n' "${CMDC}" >> "$tmpScript"
-                jfCmd="sbatch ${SBATCH_ARGS} --mem 8GB --cpus-per-task 1 -o $WORKDIR/jungfrau_${EXP}_RUN${RUN}_%J.out $tmpScript"
-                echo "run in queue: $jfCmd"        
-                SUBMISSION=$($jfCmd)
-                check_for_submit_error
-                echo "$SUBMISSION"
-                THISJOBID=$(echo "$SUBMISSION" | awk '{print $4}')
-                JOBIDS+=( "$THISJOBID" )
-                NJOBS=$((NJOBS+1))
+                for ((segment=0; segment<$NSEG; segment++)); do
+		    if [ $NSEG -lt 99 ]; then
```

Comment:
Are there cases where NSEG will remain at 99?

Reply:
Yes, if people don't contain the size of the jungfrau/epix in their alias.